docs/src/modules/concepts/pages/multi-region.adoc

```diff
@@ -16,10 +16,30 @@ Multi-region operations are ideal for:
 * Low latency global reads
 * Low latency global writes
 
+Akka has two replication modes; replicated reads and replicated writes.
+
 == Replicated reads
 
 Akka's replicated reads offers full data replication across regions and even cloud providers, without any changes to the service implementation: an entity has its "home" in one _primary region_, while being replicated to multiple other regions.
 
+Read requests are always handled locally within the region that they occur.
+
+An entity can only be written within a single region, known as the primary region.
+
+How Akka assigns the primary region to an entity is configurable. There are two modes for primary selection: **pinned-region-writes** and **request-region-writes**.
```

Comment:
We must explain the primary selection modes first, before presenting the example. And the current example is only valid for pinned writes.